### PR TITLE
README Updates

### PR DESCRIPTION
**Pull request by sarahmcdougall**
# Summary
Updates the README based on new changes from the design overhaul, adding a custom resource, automatic calculation, etc.

## New Behavior
None

## Code Changes
None

# Testing Guidance
It will be easier to examine the README by opening it as a markdown rather than looking at the code diff. The changes reflect recent PRs (some of which that are not merged), so those should get merged before this README update. Compare README changes to the new features by running `npm run dev` and comparing the new features with the documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,31 +61,37 @@ FQM-Testify is a React-based web application for analyzing FHIR-based electronic
...
 
 ### Running Measure Calculation on a Test Case
 
-FQM-Testify allows for running measure calculation on a single test case for a given measure by selecting a patient entry and clicking the calculate button. Clicking the calculate button will display logic highlighting using [the fqm-execution library](https://github.com/projecttacoma/fqm-execution). The measure calculation can be regenerated if non-patient test resources are added by clicking the calculate button. The logic highlighting can be hidden and shown using the hide/show logic highlighting button next to the calculate button.
+FQM-Testify allows for running measure calculation on a single test case for a given measure. Selecting a patient entry triggers calculation and displays logic highlighting using [the fqm-execution library](https://github.com/projecttacoma/fqm-execution) in the right panel. The measure calculation automatically regenerates if non-patient test resources are added to the patient.
```
Comment (hossenlopp) [inline README.md:90]: They recalculate on change as well, not just new resource creation.

**Review by hossenlopp — COMMENTED** [review]
One small comment on this.

File: README.md [hunk 2]
```diff
@@ -61,31 +61,37 @@ FQM-Testify is a React-based web application for analyzing FHIR-based electronic
...
 
-FQM-Testify allows the user to create non-Patient FHIR resources after at least one Patient resource is created in or imported to the app. Once a Measure Bundle is uploaded and a Patient is created, FQM-Testify will generate a list of non-Patient resource templates based on the FHIR `dataRequirements` of the uploaded measure (`dataRequirements` calculations and all other FHIR eCQM calculation in the app is done using [the fqm-execution library](https://github.com/projecttacoma/fqm-execution)). New non-Patient resources can be created by clicking on a test Patient, then clicking on a non-Patient resource from the generated list. This will open a code editor modal pre-populated with an appropriate test resource that references the selected Patient. Non-Patient resources can be examined, edited, and deleted by clicking on the Patient resource they are associated with.
+FQM-Testify allows the user to create non-Patient FHIR resources after at least one Patient resource is created in or imported to the app. To create a FHIR resource for a test patient, first select the test patient from the left panel. Once selected, FHIR resources can be created from the middle panel in two ways.
```
Comment (elsaperelli) [inline README.md:76]:
```suggestion
FQM-Testify allows the user to create non-patient FHIR resources after at least one patient resource is created in or imported to the app. To create a FHIR resource for a test patient, first select the test patient from the left panel. Once selected, FHIR resources can be created from the middle panel in two ways.
```
Not sure if you want these capitalized

File: README.md (hunk 1 above)
Comment (elsaperelli) [inline README.md:90]: You can also add in that it automatically generates when the patient is edited or when a test resource is edited! Also if the measure bundle and measurement period are changed.

**Review by elsaperelli — CHANGES_REQUESTED** [review]

**Review by elsaperelli — APPROVED** [review]
Great! Way to keep up with the documentation :)

**Review by hossenlopp — APPROVED** [review]

**Comment by hossenlopp** [comment]
Just needs to be rebased before merging.

**Pull request merged**